README.md

```diff
@@ -1,13 +1,16 @@
-# Project Express API
+# project-express-api
 
-Replace this readme with your own information about your project.
+This is my first backend project at Technigo. Creating API from row json file and making endpoints. 
+I didn't have much issues with making API, yet I got a bit stuck with find()/filter() methods, and finding a way to make query/params works.
+The most issue I encountered with this project was deploying backend/frontend.
+I created both in one folder, and it was quite confusing as I have never done that before. While I was deploying API with renderer, I got so many errors which I didn't get when I was building API in local host.
+So that was really confusing, and I thought that was the problem of settings. So I was changing possible settings options for hours, but it didn't fix. So I looked up package.json and trying to find a solution on google.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I found out that the errors came from my dependencies and their versions. I deleted few of them and degraded my Node (it was the newest version), then it started work. My deploy was ok, yet it got errors. This was from replaceAll(), and it is a new methods,
+so under Node 15, doesn't support it. So I changed it replace with regex, and it started to work!! Finally.
 
-## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If I could do it again, I would work more with error handlings. As my knowdlege of Node.js/express is not much, I could not do a lot of them. So definetly, I would like to add that in the future.
 
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+API:https://my-first-api-91k4.onrender.com/
+Deployed site:https://cosmic-strudel-6f1e10.netlify.app/
```

Comment:
Please check this site, for me it did not work. (API works, Deployed site not working)

Reply:
Thank you for letting me know!! I put the wrong one!